protostar/starknet_gateway/starknet_request.py

```diff
@@ -8,7 +8,7 @@
 
 @dataclass
 class StarknetRequest:
-    Action = Literal["DEPLOY", "DECLARE"]
+    Action = Literal["DEPLOY", "DECLARE", "CALL"]
```

Comment:
Is this in scope of this PR?

Reply:
Resolving: Moved it to the Call Cheatcode PR